### PR TITLE
UIText hover show outline

### PR DESCRIPTION
**Pull request by rrandom**
## Related issue/feature
#50 
## Description of the changes proposed in the pull request
Add event listener for mouseover and mouseout
## The person responsible for reviewing proposed changes
@TyrealGray

### CODE REVIEW

**Comment by rrandom** [comment]
Suggestion: The project should use ESLint.

File: LOMS/src/render/uiText.js [hunk 1]
```diff
@@ -17,6 +17,15 @@ export default class UIText extends Actor {
             this._sprite.mousedown = (e) => {
                 this._onClick(e);
             }
+            this._sprite.mouseover = (e) => {
+                this._sprite.style = Object.assign({}, this._style, {
```
Comment (TyrealGray) [inline LOMS/src/render/uiText.js]: Thank you for creating this PR, LOMS support ES6 , so maybe using spread operator? :)
You can find example [here](https://medium.com/@corinnemariekelly/object-assign-vs-spread-operator-577c889dbadc)

**Comment by TyrealGray** [comment]
> Suggestion: The project should use ESLint.

Good point, I use WebStorm so the IDE do the job already.  I guess other IDE might not show hint automatically.

But a strict rule also would be annoying, is there any minimal set for ESLint?

**Comment by rrandom** [comment]
Yup, but considering the project as a cooperation work, the OS is various and the IDE is various. People will submit different style of code as the project grows which will mess up the git history. For myself, I'm using [airbnb](https://github.com/airbnb/javascript) style.

**Comment by TyrealGray** [comment]
@rrandom Are you in QQ group as well? We clearly should think about it. :)

**Comment by rrandom** [comment]
no. add me in plz

File: LOMS/src/render/uiText.js [hunk 2]
```diff
@@ -17,6 +17,16 @@ export default class UIText extends Actor {
             this._sprite.mousedown = (e) => {
                 this._onClick(e);
             }
+            this._sprite.mouseover = (e) => {
+                this._sprite.style = { ...this._style, ...{
+                        stroke: 'white',
+                        strokeThickness: 5,
+                    }
+                };
+            }
```
Comment (TyrealGray) [inline LOMS/src/render/uiText.js]: should be this way `this._sprite.style = { ...this._style, 
                        stroke: 'white',
                         strokeThickness: 5
                 };`

Comment (rrandom) [reply LOMS/src/render/uiText.js]: fixed

**Pull request merged**